mbrl/models/gaussian_mlp.py

```diff
@@ -124,6 +136,12 @@ def create_linear_layer(l_in, l_out):
 
         self._propagation_indices: torch.Tensor = None
 
+    def _get_activation_func(self,activation_name, activation_params):
+        activation_func = getattr(nn,activation_name)
```

Comment:
There is a lint error here (no space after a comma). I got a message asking about some issues with pre-commit, but I couldn't see it in the repo. Were you able to solve them? Let me know if you have any questions about this.

Reply:
Yes, I could solve the issues with pre-commit!.
![image](https://user-images.githubusercontent.com/43511383/131226974-0088c64f-974d-4314-a350-18b07402a713.png)